docs/starter-web.md

```diff
@@ -544,30 +544,27 @@ If your service is required to support the metadata context but is not intereste
 continue here:
 
 Services that use the sda-spring-boot-commons:
+
 - can access the current `org.sdase.commons.spring.boot.metadata.context.MetadataContext` in their
   implementation
-- will automatically load the context from incoming HTTP requests into the thread handling the
+- will automatically load the context from incoming HTTP requests into the thread, handling the
   request, if you register `org.sdase.commons.spring.boot.web.metadata.MetadataContextConfiguration`
 - will automatically load the context from consumed Kafka messages into the thread handling the
   message and the error when handling the message fails when the consumer is configured with one of
   the provided `org.sdase.commons.spring.boot.kafka.SdaKafkaConsumerConfiguration`
-- will automatically propagate the context to other services via HTTP when using a platform client
-  that uses
-  the `org.sdase.commons.spring.boot.web.metadata.MetadataContextClientConfiguration`
-  configuration, e.g:
-  - ```java
-    @FeignClient(
+- will automatically propagate the context to other services via HTTP when using
```

Comment:
May we keep the 2 examples? One with the raw FeignClient and the other one with the PlatformClient?